Commited for 2.33

commit a6c9f54a3471a527ff502f26e3a1c1b03ab39d6e
Author: Samuel Thibault <samuel.thibault@ens-lyon.org>
Date:   Sun Dec 13 13:29:25 2020 +0000

    hurd: Add __libc_open and __libc_close
    
    Needed by libpthread for sem_open and sem_close

--- a/hurd/Versions
+++ b/hurd/Versions
@@ -160,6 +160,7 @@
     __lll_abstimed_lock; __lll_robust_lock;
     __lll_robust_abstimed_lock; __lll_robust_trylock;
     __lll_robust_unlock;
+    __libc_open; __libc_close;
 
     # Used by libpthread.
     _hurd_sigstate_set_global_rcv;
--- a/sysdeps/mach/hurd/close.c
+++ b/sysdeps/mach/hurd/close.c
@@ -31,4 +31,5 @@
   return err ? __hurd_fail (err) : 0;
 }
 libc_hidden_def (__close)
+strong_alias (__close, __libc_close)
 weak_alias (__close, close)
